ProductsService/ProductsService.Infrastructure/Data/Configuration/CategoryConfiguration.cs

```diff
@@ -0,0 +1,181 @@
+            {
+                Id = 1,
+                Name = "Smartphones",
+            },
+            new Category
+            {
+                Id = 2,
+                Name = "Laptops",
+            },
+            new Category
+            {
+                Id = 3,
+                Name = "Brand",
+                ParentId = 1,
+            },
+            new Category
+            {
+                Id = 4,
+                Name = "Brand",
+                ParentId = 2,
+            },
+            new Category
+            {
+                Id = 5,
+                Name = "RAM",
+                ParentId = 1,
+            },
+            new Category
+            {
+                Id = 6,
+                Name = "RAM",
+                ParentId = 2,
+            },
+            new Category
+            {
+                Id = 7,
+                Name = "ROM",
+                ParentId = 1,
+            },
+            new Category
+            {
+                Id = 8,
+                Name = "ROM",
+                ParentId = 2,
+            },
+            new Category
+            {
+                Id = 9,
+                Name = "Processor",
+                ParentId = 2,
+            },
+            new Category
+            {
+                Id = 10,
+                Name = "Samsung",
+                ParentId = 3,
+            },
+            new Category
+            {
+                Id = 11,
+                Name = "Apple",
+                ParentId = 3,
+            },
+            new Category
+            {
+                Id = 12,
+                Name = "12GB",
+                ParentId = 5,
+            },
+            new Category
+            {
+                Id = 13,
+                Name = "8GB",
+                ParentId = 5,
+            },
+            new Category
+            {
+                Id = 14,
+                Name = "512GB",
+                ParentId = 7,
+            },
+            new Category
+            {
+                Id = 15,
+                Name = "Apple",
+                ParentId = 4,
+            },
+            new Category
+            {
+                Id = 16,
+                Name = "16GB",
+                ParentId = 6,
+            },
+            new Category
+            {
+                Id = 17,
+                Name = "18GB",
+                ParentId = 6,
+            },
+            new Category
+            {
+                Id = 18,
+                Name = "512GB",
+                ParentId = 8,
+            },
+            new Category
+            {
+                Id = 19,
+                Name = "1024GB",
+                ParentId = 8,
+            },
+            new Category
+            {
+                Id = 20,
+                Name = "M3",
+                ParentId = 9,
+            },
+            new Category
+            {
+                Id = 21,
+                Name = "M3 Pro",
+                ParentId = 9,
+            }
+        );
```

Comment:
what do you think about moving all data seeding from all configuration classes to a separate class (DbInitializer or smth like that?)

Reply:
Vasya told me to use SeedData instead of DbInitializer, if it's neccessary, I can move it into initializer class